backend/tests/test_15_1_charts.py

```diff
@@ -73,7 +73,7 @@ async def test_get_aggregated_jmp_chart_data(
                 {"administration": 25, "child": []},
                 {"administration": 26, "child": []},
             ],
-            "total": 4,
+            "total": 2,
```

Comment:
@ifirmawan are there any reason why the value changed?

Reply:
Hi mas @dedenbangkit, yes because the endpoint applies to the filter by the administration. 
https://github.com/akvo/wai-sdg-portal/pull/351/files/39f40a4f4cd1499845f921c65f63e1703df341fc#diff-7bbd6aa165f2171dac9ba4daa6ca7ad4c32b88e7f97c2f7bb1154b1b9ba65559R164-R171